src/media/mline.rs

```diff
@@ -414,6 +414,15 @@ impl MLine {
         };
         buf.truncate(header_len + body_len);
 
+        #[cfg(feature = "log_stats")]
+        if let Some(delay) = next
+            .body
+            .queued_at()
+            .map(|i| Instant::now().duration_since(i))
```

Comment:
The `now` passed in as argument seems more appropriate since that would let us control time externally.

Reply:
Ah, I didn't realise we had `now` in this context. Will change